guide/14-deep-learning/how-sequencetosequence-works.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p><code>SequenceToSequence</code> class to see the <s>available </s>supported backbones</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/869/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>fixed</p>